### ISSUE TITLE
installer-debian vs debian-installer

### ISSUE THREAD

**Issue by kareltucek** [unindented/grunt-electron-debian-installer#15]
Hi, could you please fix the word order in all the examples? This is a very nasty thing to debug for a person who knows almost nothing about grunt...

**Comment by unindented** [comment]
@kareltucek I've searched the codebase for `debian-installer` and can't find anything. Could you point to the specific examples?

**Comment by kareltucek** [comment]
Now when you are saying it, I must admit you are right. I was actually directed to this repository from https://www.npmjs.com/package/grunt-electron-debian-installer and must have assumed that the page's content and this readme's content were the same ones. (Of course I did not notice that there was another page for the correct version and considered the old npm page being an up-to-date one) 

Anyway the fact that the out-of-date npm page contains an explicit mention about the rename suggests that its content may have been updated to reflect the rename - some explicit warning about this not being true and a link to the correct one may save some future visitors from the same confusion. 

(Well - all was said - feel free to close this)
